.drone.yml

```diff
@@ -4656,6 +4656,55 @@ volumes:
 - name: dockersock
   temp: {}
 
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/misc.go:30
+################################################
+
+kind: pipeline
+type: kubernetes
+name: publish-apt-rfd0058-repos
+trigger:
+  event:
+    include:
+    - promote
+  target:
+    include:
+    - production
+  repo:
+    include:
+    - gravitational/*
+workspace:
+  path: /go
+clone:
+  disable: true
+steps:
+- name: Publish debs to APT repos
+  image: ubuntu:20.04
```

Comment:
In other pipelines we use either `docker` or `docker:git` images. Any reason you chose `ubuntu:20.04` here?

Reply:
In this case there is no need for docker in docker here - just need go and aptly. That being said, I need to update this to `golang:1.18.1-bullseye`.